llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -554,8 +554,10 @@ void RISCVFrameLowering::emitPrologue(MachineFunction &MF,
   if (RVFI->isPushable(MF) && FirstFrameSetup != MBB.end() &&
       FirstFrameSetup->getOpcode() == RISCV::CM_PUSH) {
     // Use available stack adjustment in push instruction to allocate additional
-    // stack space.
-    uint64_t Spimm = std::min(StackSize, (uint64_t)48);
+    // stack space. Align the stack size down to a multiple of 16. This is
+    // needed for RVE.
```

Comment:
I may miss something here, why is this needed for RVE? Should all be 16-bytes aligned?

Reply:
RVE stack alignment is XLen-bytes not 16-bytes